.circleci/config.yml

```diff
@@ -194,6 +193,63 @@ jobs:
       - store_artifacts:
           path: /tmp/test-artifacts/<< parameters.suite >>.json
 
+
+  test-race:
```

Comment:
This should also be added to template.yml - we generate this config from there (with `make circleci`)